android.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+#
+# Replace jdk.table xml file
+# For Android Studio to detect the correct JDK path
+# we need to preset the path in the jdk.table.xml
+#
+
+OPTIONS_DIR="$HOME/.AndroidStudio2.2/config/options"
+
+if [[ ! -f ${OPTIONS_DIR}/jdk.table.xml ]]; then
+  mkdir -p ${OPTIONS_DIR}
+  cp /app/AndroidStudioConfig/jdk.table.xml ${OPTIONS_DIR}/jdk.table.xml
+fi
+
+exec /app/bin/studio.sh $@
```

Comment:
Please use double quotes around `$@` to avoid issues with filenames and arguments which may contains spaces (or other characters used by the shell to split arguments):

```sh
exec /app/bin/studio.sh "$@"
```